code/datums/components/overlay_lighting.dm

```diff
@@ -277,6 +278,8 @@
 ///Called when the current_holder is qdeleted, to remove the light effect.
 /datum/component/overlay_lighting/proc/on_holder_qdel(atom/movable/source, force)
 	SIGNAL_HANDLER
+	if(QDELETED(current_holder))
+		return
```

Comment:
what for?

Reply:
due to signal queues, dying as a holopara owner sends a signal to kill the holopara, which then kills this component because the atom is deleted, which sets current_holder to null, so when this runs due to the signal queue it runtimes
rohesie who made the system originally said it was fine